recipes/trexio/meta.yaml

```diff
@@ -0,0 +1,58 @@
+
+{% set name = "trexio" %}
+{% set version = "2.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/TREX-CoE/trexio/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 232866c943b98fa8a42d34b55e940f7501634eb5bd426555ba970f5c09775e83
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+    - autoconf  
+    - automake  
+    - libtool   
+    - make      
+    - gnuconfig 
+    - pkg-config
+
+  host:
+    - hdf5
+    # the dependencies below are not required by TREXIO but they get exported from HDF5
+    # and if they are not listed here than the `conda-build --error-overlinking` fails
+    - zlib    
+    - openssl 
+    - libcurl 
+
+  run:
+    - hdf5 
+
+test:
+  commands:
+    - test -f $PREFIX/lib/libtrexio.a  # [unix]
```

Comment:
Please check for all the files that you are shipping. You should be shipping a `shared` library and header files instead of a static library with no headers.

Reply:
Done.